src/bepasty/tests/test_rest_server.py

```diff
@@ -224,7 +224,7 @@ def test_config(client_fixture):
 
 
 def _upload(client, data, token=None, filename=None, ftype=None, lifetime=None,
-            range_str=None, trans_id=None, no_encode=False):
+            range_str=None, trans_id=None, no_encode=False, no_range=False):
```

Comment:
again: avoid double negation. `range=True`

Reply:
fixed, changed to set_range

Reply:
ah, right, better not overwrite useful builtins.